src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySaved/Components/OpportunitySaved.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import * as S from '../styles';
+import universityOfMelbourne from '../../../ignitus-Assets/ignitus-Images/img-Png/universityOfMelbourne.png'
+import { Description } from '../types';
+import { Data, Title, Subtitle, OpportunityDescription } from '../constants';
+import {withErrorBoundary} from '../../../../ignitus-ErrorHandlingComponents/errorBoundary';
+
+export const Opportunity = ({title, description, subtitle, data}: Description) => (
+  <S.OpportunityContainer>
+    <S.ImageDescriptionContainer>
+      <S.ImageContainer>
+        <img src={universityOfMelbourne} alt={title} height="120" />
+      </S.ImageContainer>
+      <S.DescriptionContainer>
+        <S.OpportunityTitle>{title}</S.OpportunityTitle>
+        <S.OpportunitySubTitle>{subtitle}</S.OpportunitySubTitle>
+          <S.OpportunityDescription>
+            {description}
+          </S.OpportunityDescription>
+      </S.DescriptionContainer>
+    </S.ImageDescriptionContainer>
+  </S.OpportunityContainer>
+);
+
+const opportunity = [{'title':Title[0],'description':OpportunityDescription[0],'subtitle':Subtitle[0], 'data':Data[0]},
```

Comment:
You need to add this single card in` Card Layout` http://www.ignitus.org/interface/cardLayout as well.